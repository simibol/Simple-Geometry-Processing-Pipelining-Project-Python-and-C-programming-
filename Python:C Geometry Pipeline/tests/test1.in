1.0q, 2.00
3.e0, 5.00
4.00, 6.00
6.00, -6.00
8.53, -5.82