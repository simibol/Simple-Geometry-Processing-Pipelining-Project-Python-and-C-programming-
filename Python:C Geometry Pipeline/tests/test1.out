read 5 points
4.00, 6.00
6.00, -6.00
8.53, -5.82
This is a triangle